dhis2_android_maps/src/main/java/org/dhis2/maps/model/MapSelectorScreenState.kt

```diff
@@ -15,8 +15,23 @@ data class MapSelectorScreenState(
     val displayPolygonInfo: Boolean,
     val locationState: LocationState,
     val isManualCaptureEnabled: Boolean,
+    val forcedLocationAccuracy: Int,
 ) {
-    val doneButtonEnabled = selectedLocation !is SelectedLocation.None && !captureMode.isSwipe()
+    private fun getDoneButtonEnabledState(): Boolean {
+        return when {
+            (forcedLocationAccuracy == -1) -> {
```

Comment:
What do you think about putting the -1 in a constant variable to provide better context to the person reading the code?

Reply:
good idea! I will update the PR